The unit testing of tensor_test failed.

I run the unit testing for develop branch on my GPU server and did not change any code. The tensor_test failed.
```
[luotao@m1-idl-gpu2-bak24: build] $ make test ARGS='-R tensor_test -V'   
Running tests...
UpdateCTestConfiguration  from :/home/luotao/Paddle/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/home/luotao/Paddle/build/DartConfiguration.tcl
Test project /home/luotao/Paddle/build
Constructing a list of tests
Done constructing a list of tests
Checking test dependency graph...
Checking test dependency graph end
test 88
    Start 88: tensor_test

88: Test command: /home/luotao/Paddle/build/paddle/framework/tensor_test
88: Test timeout computed to be: 9.99988e+06
88: Running main() from gtest_main.cc
88: [==========] Running 6 tests from 1 test case.
88: [----------] Global test environment set-up.
88: [----------] 6 tests from Tensor
88: [ RUN      ] Tensor.Dims
88: [       OK ] Tensor.Dims (0 ms)
88: [ RUN      ] Tensor.DataAssert
88: unknown file: Failure
88: C++ exception with description "basic_string::_S_construct null not valid" thrown in the test body.
88: [  FAILED  ] Tensor.DataAssert (0 ms)
88: [ RUN      ] Tensor.MutableData
88: [       OK ] Tensor.MutableData (311 ms)
88: [ RUN      ] Tensor.ShareDataWith
88: unknown file: Failure
88: C++ exception with description "basic_string::_S_construct null not valid" thrown in the test body.
88: [  FAILED  ] Tensor.ShareDataWith (1 ms)
88: [ RUN      ] Tensor.Slice
88: [       OK ] Tensor.Slice (0 ms)
88: [ RUN      ] Tensor.CopyFrom
88: [       OK ] Tensor.CopyFrom (0 ms)
88: [----------] 6 tests from Tensor (312 ms total)
88: 
88: [----------] Global test environment tear-down
88: [==========] 6 tests from 1 test case ran. (312 ms total)
88: [  PASSED  ] 4 tests.
88: [  FAILED  ] 2 tests, listed below:
88: [  FAILED  ] Tensor.DataAssert
88: [  FAILED  ] Tensor.ShareDataWith
88: 
88:  2 FAILED TESTS
1/2 Test #88: tensor_test ......................***Failed    0.46 sec
Total Test time (real) =   0.48 sec
The following tests FAILED:
         88 - tensor_test (Failed)
Errors while running CTest
make: *** [test] Error 8
```

--- a/paddle/platform/enforce.h
+++ b/paddle/platform/enforce.h
@@ -25,10 +25,6 @@ limitations under the License. */
 #include "paddle/string/printf.h"
 #include "paddle/string/to_string.h"
 
-#ifdef __GNUC__
-#include <cxxabi.h>  // for __cxa_demangle
-#endif
-
 #ifndef PADDLE_ONLY_CPU
 
 #include "paddle/platform/dynload/cublas.h"
@@ -45,19 +41,6 @@ limitations under the License. */
 
 namespace paddle {
 namespace platform {
-
-namespace {
-#ifdef __GNUC__
-inline std::string demangle(std::string name) {
-  int status = -4;  // some arbitrary value to eliminate the compiler warning
-  std::unique_ptr<char, void (*)(void*)> res{
-      abi::__cxa_demangle(name.c_str(), NULL, NULL, &status), std::free};
-  return (status == 0) ? res.get() : name;
-}
-#else
-inline std::string demangle(std::string name) { return name; }
-#endif
-}
 
 struct EnforceNotMet : public std::exception {
   std::exception_ptr exp_;
@@ -79,7 +62,7 @@ struct EnforceNotMet : public std::exception {
       Dl_info info;
       for (int i = 0; i < size; ++i) {
         if (dladdr(call_stack[i], &info)) {
-          auto demangled = demangle(info.dli_sname);
+          auto demangled = info.dli_sname;
           auto addr_offset = static_cast<char*>(call_stack[i]) -
                              static_cast<char*>(info.dli_saddr);
           sout << string::Sprintf("%-3d %*0p %s + %zd\n", i,

